modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry.
+
+.Prerequisites
+
+//There are instances when you want to use cert-manager instead of general certificates or default certificates. OpenShift Service Mesh requires cert-manager to provide:
+
+* A TLS secret with an intermediate CA key and certificate for `istiod`:
+[source, yaml]
+----
+name: istiod-tls;
+namespace: the same one in which SMCP is deployed;
+data keys: tls.key and tls.crt.
+----
+
+* A ConfigMap with a root certificate:
+[source, yaml]
+----
+name: istio-ca-root-cert or custom specified in spec.security.certificateAuthority.cert-manager.rootCAConfigMapName;
+namespace: the same one in which SMCP is deployed;
+data key: root-cert.pem.
+----
+
+Fore more information on creating a Kubernetes TLS secret and a Kubernetes ConfigMap with root certificate, see Additional resources.
+
+[NOTE]
+====
+Currently, `istio-csr` does not allow customizing the namespace in which secrets and config maps are created. 
```

Comment:
This note is no longer truth. Jetstack released recently istio-csr v0.6.0, which supports namespace selector, so in this note I would recommend users to set `app.controller.configmapNamespaceSelector: "maistra.io/member-of: <istio-namespace>` to avoid creation config maps in all namespaces.